AppSettingsManagerUI/src/app/home/view-settings/view-settings.component.scss

```diff
@@ -0,0 +1,181 @@
+.vertical-nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 150px;
+  background-color: hsl(0, 0%, 27%);
+  padding: 10px 0;
+  display: flex; /* Add this line to use flexbox */
+  flex-direction: column; /* Add this line to stack items vertically */
+  justify-content: center; /* Add this line to center items vertically */
+}
+
+.vertical-nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.vertical-nav li {
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.nav-title {
+  font-size: 25px;
+  font-weight: bold;
+  color: rgb(191, 139, 215);
+  text-align: center;
+  border: 1px solid white;
+  padding: 10px;
+  margin-bottom: 20px;
+  position: absolute;
+  top: 10px;
+  left: 13px;
+  font-family: Plus Jakarta Sans;
+  font-weight: 100;
+}
+
+.nav-create {
+  margin-bottom: auto;
+  align-self: center; /* Add this line to center the button horizontally */
+}
+
+.nav-logout {
+  position: absolute;
+  bottom: 50px;
+  left: 32px;
+}
+
+.mat-header-cell-centered {
+  text-align: center;
+}
+
```

Comment:
Were you able to position the header attributes such Id and Version so that they can be seen? I remember Id being hidden under the vertical navbar.